### ISSUE TITLE
Regarding Data Augumentation

### ISSUE THREAD

**Issue by aditya7874** [davidsandberg/facenet#803]
Hi there,
            As far as I understood, while training using softmax the data augmentation is being done in the function facenet.create input pipeline. Based on the input args we are modifying the image by applying the required functions like flipping and we are appending to a list which is passed to tf.train.batch_join to get an image batch. My doubt here is we are appending only the modified image we are not appending the original image i.e we are training with only modified images
 
For cropping it is ok because the input size to inception architecture is fixed. But for flipping shouldn't it be like both original and flipped should be added to training list.
Can anyone give me an example of data augmentation while training instead of offline in tensorflow.
